fuzzers/libfuzzer/fuzzer.py

```diff
@@ -63,6 +63,9 @@ def run_fuzzer(input_corpus, output_corpus, target_binary, extra_flags=None):
         # Don't use LSAN's leak detection. Other fuzzers won't be using it and
         # using it will cause libFuzzer to find "crashes" no one cares about.
         '-detect_leaks=0',
+
+        # Store crashes along with corpus for bug based benchmarking.
+        f'-artifact_prefix={output_corpus}/',
```

Comment:
This might have some weird effects because libFuzzer reloads its corpus directories (maybe this one wont be reloaded because it is an output corpus).
Maybe the right approach is to put output corpora dirs in a subdirectory of output_corpus? WDYT?

Reply:
Fixed.